lib/rules/declaration-block-no-redundant-longhand-properties/index.mjs

```diff
@@ -187,6 +187,9 @@ const haveConflicts = [
 	'border-right',
 	'border-bottom',
 	'border-left',
+	'grid-area',
```

Comment:
[suggestion] Even if this `grid-area` is absent, the test passes. Is it unnecessary?

```suggestion
```

Reply:
You're right. I’ve fixed it. Thanks for pointing it out!
[fix: remove unnecessary props](https://github.com/stylelint/stylelint/pull/8023/commits/021c31a11493843637dd7e27315962152ac37eae)